src/engine/InputManager.java

```diff
@@ -86,16 +101,21 @@ public void keyReleased(final KeyEvent key) {
 	 * @return True if the key was double-tapped, false otherwise.
 	 */
 	public boolean isDoubleTap(int keyCode) {
+		if (keyCode != KeyEvent.VK_LEFT && keyCode != KeyEvent.VK_RIGHT) { return false; }
```

Comment:
It makes player 2 can't use double tap feature
And it will not works in future when ship's movement key gonna be changed.